ablySpec/RealtimeClientChannel.swift

```diff
@@ -550,6 +550,70 @@ class RealtimeClientChannel: QuickSpec {
 
             }
 
+            // RTL7
+            context("subscribe") {
+
+                // RTL7d
+                pending("should deliver the message even if there is an error while decoding") {
+
+                    for cryptoTest in [CryptoTest.aes128, CryptoTest.aes256] {
+                        it("using \(cryptoTest) ") {
+                            let options = AblyTests.commonAppSetup()
+                            options.autoConnect = false
+                            let client = ARTRealtime(options: options)
+                            client.setTransportClass(TestProxyTransport.self)
+                            client.connect()
+                            defer { client.close() }
+
+                            let (keyData, ivData, messages) = AblyTests.loadCryptoTestData(cryptoTest)
+                            let testMessage = messages[0]
+
+                            let cipherParams = ARTCrypto.defaultParamsWithKey(keyData, iv: ivData)
+                            let channelOptions = ARTChannelOptions(encrypted: cipherParams)
+                            let channel = client.channels.get("test", options: channelOptions)
+
+                            let transport = client.transport as! TestProxyTransport
+
+                            transport.beforeProcessingSentMessage = { protocolMessage in
+                                if protocolMessage.action == .Message {
+                                    expect(protocolMessage.messages![0].data as? String).to(equal(testMessage.encrypted.data))
+                                    expect(protocolMessage.messages![0].encoding).to(equal(testMessage.encrypted.encoding))
```

Comment:
Failing because:
 - `expected to equal <utf-8/cipher+aes-128-cbc/base64>, got <cipher+aes-128-cbc/base64>`
 - `expected to equal <utf-8/cipher+aes-256-cbc/base64>, got <cipher+aes-256-cbc/base64>`